mono/metadata/custom-attrs.c

```diff
@@ -1884,9 +1884,10 @@ mono_reflection_get_custom_attrs_info_checked (MonoObjectHandle obj, MonoError *
  * of that type are returned. The objects are fully build. Return NULL if a loading error
  * occurs.
  */
-MonoArray*
```

Comment:
If the declaration in the `.h` file is marked as `MONO_API` then there is no need to do it in the `.c` file too.